packages/@sanity/form-builder/src/inputs/PortableText/text/TextBlock.styles.ts

```diff
@@ -99,8 +99,9 @@ function textBlockStyle(props: TextBlockStyleProps & {theme: Theme}) {
 
 export const TextRoot = styled.div<TextBlockStyleProps>(textBlockStyle)
 
-export const TextBlockFlexWrapper = styled(Flex)`
+export const TextBlockFlexWrapper = styled(Box)`
```

Comment:
This piece of code looks so much like an oversight to me now ("why is it a box when it clearly should have been a flex?"). Would be great to add a comment here so it doesn't accidentally gets changed back to Flex. Otherwise good!

Reply:
Added comment about this now.